spec.bs

```diff
@@ -1704,18 +1711,17 @@ To <dfn>convert a modifier to a string</dfn> given a [=part/modifier=] |modifier
   To <dfn>canonicalize a protocol</dfn> given a string |value|:
 
   1. If |value| is the empty string, return |value|.
-  1. Let |dummyURL| be a new [=URL record=].
-  1. Let |parseResult| be the result of running the [=basic URL parser=] given |value| followed by "`://dummy.test`", with |dummyURL| as <i>[=basic URL parser/url=]</i>.
+  1. Let |parseResult| be the result of running the [=basic URL parser=] given |value| followed by "`://dummy.invalid/`".
      <p class="note">Note, [=basic URL parser/state override=] is not used here because it enforces restrictions that are only appropriate for the {{URL/protocol}} setter.  Instead we use the protocol to parse a dummy URL using the normal parsing entry point.</p>
   1. If |parseResult| is failure, then throw a {{TypeError}}.
-  1. Return |dummyURL|'s [=url/scheme=].
+  1. Return |parseResult|'s [=url/scheme=].
 </div>
 
 <div algorithm>
   To <dfn>canonicalize a username</dfn> given a string |value|:
 
   1. If |value| is the empty string, return |value|.
-  1. Let |dummyURL| be a new [=URL record=].
+  1. Let |dummyURL| be the result of [=create a dummy URL=].
```

Comment:
You want "creating" not "create" here (and below) to get a proper sentence (you don't have to change the `<dfn>` for that, as I mentioned Bikeshed takes care of making that work).

Reply:
My bad, fixed.